packages/mojatax/src/Utils.js

```diff
@@ -0,0 +1,87 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  assertRelativeUrl,
+  logResponse,
+} from '@openfn/language-common/util';
+import nodepath from 'node:path';
+
+export const authorize = state => {
+  const auth = state.configuration;
+  if (auth.access_token) {
+    return state;
+  }
+  const clientId = auth.clientId;
+  const password = auth.password;
+  const headers = {};
+
+  if (clientId && password) {
+    Object.assign(headers, {
+      'Content-Type': 'application/json',
+    });
+
+    const body = {
+      client_id: clientId,
+      password: password,
+    };
+    const options = {
+      body: JSON.stringify(body),
```

Comment:
Are you 100% sure we need to stringify this? I'm sure the common helper should do it for you